projects/components/src/select/select.component.ts

```diff
@@ -115,6 +122,10 @@ export class SelectComponent<V> implements AfterContentInit, OnChanges {
 
   public ngAfterContentInit(): void {
     this.selected$ = this.buildObservableOfSelected();
+
+    setTimeout(() => {
+      this.selectContainerWidth = `${this.selectContainer.nativeElement.offsetWidth}px`;
```

Comment:
Can we use `this.selectContainer.nativeElement.offsetWidth` directly in the template? Advantage would be that if `selectContainer`'s width changes after initial render(let's say it's width is 20% and a window resize happens) then the popover width would automatically match it, everytime

Reply:
You can do this directly in the dom itself. I played around with it an added a similar change to multi-select component which is a separate but similar component. Please take a look at this change here https://github.com/hypertrace/hypertrace-ui/pull/524/files

Reply:
Thanks, it's look far simply and code become more readable, I refactor this as it used to be in your commit